mocks/oppdrag-mock/src/main/java/no/nav/okonomi/tilbakekrevingservice/KravgrunnlagGenerator.java

```diff
@@ -29,8 +30,8 @@ public static DetaljertKravgrunnlagDto hentGrunnlag() {
         detaljertKravgrunnlag.setDatoVedtakFagsystem(konvertDato(LocalDate.of(2019, 3, 14)));
         detaljertKravgrunnlag.setEnhetAnsvarlig(ENHET);
         detaljertKravgrunnlag.setFagsystemId("10000000000000000");
-        detaljertKravgrunnlag.setKodeFagomraade("FP");
-        detaljertKravgrunnlag.setKodeHjemmel("1234239042304");
+        detaljertKravgrunnlag.setKodeFagomraade("PENAP");
```

Comment:
Her må det en bedre løsning til for å gjøre mocken smart nok til å svare FP/PENAP ettersom hvem request kommer fra

Reply:
@LarsErikDahl  og @olovholm Hvordan skiller vi at **KravgrunnlagHentDetaljRequest** kommer fra pensjon eller foreldrepenger? Hvis det ikke er mulig å finne ut av requesten, kan vi overleve med generisk respons?